### DISCUSSION TITLE
How to restart MCU through code

### DISCUSSION THREAD

**Discussion by ShiPC123** [stm32duino/Arduino_Core_STM32#2385] [Q&A] [answered]
HI ,Can I restart the microcontroller through the code? I use STM32G070CBT6

**Comment by fpistm** [comment, marked as answer]
Try `NVIC_SystemReset();`

**Reply by ShiPC123** [reply]
Thanks, I will test it now

**Reply by ShiPC123** [reply]
Thanks, this one works
